documents/operating.md

```diff
@@ -294,13 +308,35 @@ in proportion of their prorated ownership percentage.  For example, if a Corpora
 
 4. 20% of the Spoils of War shall be added to the company coffers.
 
-### Compensation
-Any compensation paid to individuals working on a project must be associated with a contract that has either been approved by both the Treasure Goblin and the Overmind.
+### Project Compensation and Contracts
+Any compensation paid to individuals working on a project must be associated with a contract that has been approved by the comptroller of that project, and signed by the Overmind.
+
+### Project Resources
+Project comptrollers have wide control over how the money associated with their projects can be spent.  It is their responsibility to communicate approved expenses to the Treasure Goblin so that those expenses can be paid.
+
+Licenses and assets obtained through a given project are property of the company, but the project owner shall be guaranteed as much use of it as needed (so long as they are the ones paying for it). Comptrollers are urged to try to find assets that can be used by all BIFFUD projects.
```

Comment:
"They are the ones paying for it" sounds like the comptroller personally paid for it. We mean out of their corporate budget, right?

Reply:
Yeah I'll clarify!